src/app/modules/core/manager/tenant/components/edit/manager-tenant-edit.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, OnInit } from '@angular/core'
```

Comment:
Please, rename the file, the component class, all the related files and the selector in `tenant-details` instead of `manager-tenant-details`. Most probably this component will be shared with the landlord module. Thanks.